frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.unit.spec.ts

```diff
@@ -139,24 +139,25 @@ describe("getSeriesWithColors", () => {
       expect(seriesWithColors).toEqual(expectedSeries);
     });
 
-    it("it should assign colors from column colors", () => {
+    it("should assign colors from column colors", () => {
       const seriesWithColors = getSeriesWithColors(
-        multipleSeries,
         merge(settings, {
-          series_settings: {
-            count: {
-              color: "#987654",
+          visualization_settings: {
```

Comment:
now we can use every setting in `visualization_settings` and don't have to handpick which settings we want to static viz to see because it should see all settings anyway 🤷